Sources/Rendering/OpenGL/Texture/index.js

```diff
@@ -656,8 +657,9 @@ function vtkOpenGLTexture(publicAPI, model) {
       for (let idx = 0; idx < data.length; idx++) {
         if (data[idx]) {
           const newArray = new Uint16Array(pixCount);
```

Comment:
What if you create a very large array outside the for loop and use subarray at each iteration, could you save even more time ?